lane.py

```diff
@@ -225,13 +225,11 @@ def dy(new_value):
                 yield dy(facet)
 
     @classmethod
-    def order_facet_to_database_field(cls, order_facet, work_model):
+    def order_facet_to_database_field(cls, order_facet):
         """Turn the name of an order facet into a materialized-view field
         for use in an ORDER BY clause.
-
-        :param work_model: Either MaterializedWork or
-        MaterializedWorkWithGenre.
         """
+        from model import MaterializedWorkWithGenre as work_model
```

Comment:
Why are you importing MaterializedWorkWithGenre inline? I can't see any model.py imports that would keep you from importing it once higher up in the file.

Reply:
MaterializedWorkWithGenre isn't actually defined inside model.py until SessionManager.initialize() runs. This is because I couldn't find any way of defining a SQLAlchemy class for a materialized view without having an active database connection so SQLAlchemy could see the view for itself.